src/pages/experiments/[experimentId]/plots-and-tables/frequency/index.jsx

```diff
@@ -131,7 +120,12 @@ const frequencyPlot = ({ experimentId }) => {
 
     return (
       <center>
-        <FrequencyPlot hierarchy={hierarchy} properties={properties} config={config} />
+        <FrequencyPlot
+          experimentId={experimentId}
+          hierarchy={hierarchy}
```

Comment:
maybe you can just pass cellSets and destructure it inside the component? Less props this way.

Reply:
By passing cellsets, I can check for cellsets loading or error too

Reply:
I've changed it to use `cellSets`